fv3core/examples/standalone/runfile/compile.py

```diff
@@ -40,20 +53,56 @@ def parse_args() -> Namespace:
     namelist = f90nml.read(args.data_dir + "/input.nml")
     experiment_name, is_baroclinic_test_case = get_experiment_info(args.data_dir)
     dycore_config = DynamicalCoreConfig.from_f90nml(namelist)
-    for rank in range(dycore_config.layout[0] * dycore_config.layout[1]):
-        mpi_comm = NullComm(
-            rank=rank,
-            total_ranks=6 * dycore_config.layout[0] * dycore_config.layout[1],
-            fill_value=0.0,
-        )
-        dycore, dycore_args, stencil_factory = setup_dycore(
-            dycore_config,
-            mpi_comm,
-            args.backend,
-            is_baroclinic_test_case,
-            args.data_dir,
-        )
+    if MPI is not None:
+        comm = MPI.COMM_WORLD
+        global_rank = comm.Get_rank()
+        size = comm.Get_size()
+    else:
+        comm = None
+        global_rank = 0
+        size = 1
+    sub_tiles = dycore_config.layout[0] * dycore_config.layout[1]
+    iterations = math.ceil(sub_tiles / size)
+    gt4py.config.cache_settings["root_path"] = os.environ.get("GT_CACHE_DIR_NAME", ".")
+    for iteration in range(iterations):
+        top_tile_rank = global_rank + size * iteration
+        if top_tile_rank < sub_tiles:
+            mpi_comm = NullComm(
```

Comment:
Tag that this will fail dace orchestration

Reply:
Actually this will all fail orchestration since you have to run the actual driver, so tag that up the file :)